.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: JBoss Modules CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  build-test-matrix:
+    name: JDK${{ matrix.jdk }}-${{ matrix.impl}}-${{ matrix.os }}
```

Comment:
This project is using the jboss-parent MR JAR mechanism so it should not be built using matrix.  There aren't any instructions for setting up GH actions for this on the JBoss Parent project, but it's basically the same process as it is for SmallRye Parent which is documented here: https://github.com/smallrye/smallrye-parent/#using-mr-jar-functions-with-github-actions

Reply:
Which I see you've essentially already done... let me read further.

Reply:
Oh, I see the TARGETs thing, I'll give that a go, thanks for the pointer.